assets/js/search-bar.js

```diff
@@ -102,6 +103,16 @@ function addEventListeners () {
   })
 }
 
+function setInputPlaceholder () {
+  const searchInput = qs(SEARCH_INPUT_SELECTOR)
+
+  if (isMacOS()) {
+    searchInput.setAttribute('placeholder', 'Search - ⌘K')
+  } else {
+    searchInput.setAttribute('placeholder', 'Search - ctrl K')
+  }
+}
```

Comment:
We should not change the placeholder. Let's keep it as it is right now. I think `/` is still the most convenient for most users (and also what GitHub uses).

Reply:
Just as one datapoint to the opposite, `/` is (IMO) very inconvenient on a Swedish keyboard. Looking forward to using ctrl-k in ex_doc!

Reply:
Oh, good point. You can also press `s`. Maybe we should change the placeholder to say press `s` instead? 🤔

Reply:
No problem-o!  I forgot about github as for some reason it's one of the only sites where I don't use its keyboard shortcuts.  As a Vim user I do find `/` an odd choice and since it's used for search _within_ a document whereas `k` is used to search the docs, but github is obviously good precedent!  In any event I'm happy to bring back the placeholder.  Let me know if you'd rather I replace it with `s`.

Reply:
Let's only remove the placeholder changes for now, we can revisit in another PR.

Reply:
Done!